cgmanifest.json

```diff
@@ -31,8 +31,8 @@
          "component": {
```

Comment:
>"component": { [](start = 9, length = 14)

Will remove the component since we have a local copy.

Reply:
Please keep it. It is for helping us maintain and generating the https://github.com/microsoft/onnxruntime/blob/master/ThirdPartyNotices.txt